types/snowflake-sdk/index.d.ts

```diff
@@ -163,6 +163,21 @@ export interface SnowflakeError extends SnowflakeErrorExternal {
     externalize?: () => SnowflakeErrorExternal;
 }
 
+export enum validFetchAsStringType {
+    STRING = 'STRING',
+    BOOLEAN = 'BOOLEAN',
+    NUMBER = 'NUMBER',
+    DATE = 'DATE',
+    JSON = 'JSON',
+}
+
+// The supported types are: String, Boolean, Number, Date, and JSON.
+export interface StatementOptions {
+    start?: number;
+    end?: number;
+    fetchAsString?: any[];
```

Comment:
For consistently we should type `fetchAsString` like we do [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/e1062a77f89426b6b69ca5571a0a093c7fb89ca7/types/snowflake-sdk/index.d.ts#L577).
```suggestion
    fetchAsString?: Array<'String' | 'Boolean' | 'Number' | 'Date' | 'JSON'> | undefined;
```